code/obj/decal/poster.dm

```diff
@@ -778,6 +778,12 @@
 
 			// 0 = GLASS, MEDAL 1 = GLASS OFF, MEDAL IN CASE, 2 = GLASS OFF, MEDAL GONE,
 
+			New()
+				..()
+				var/obj/item/clothing/suit/hosmedal/M = new /obj/item/clothing/suit/hosmedal(src.loc)
+				M.desc = src.desc
+				src.contents.Add(M)
```

Comment:
```suggestion
				var/obj/item/clothing/suit/hosmedal/M = new(src)
				M.desc = src.desc
```

Reply:
Mirror these changes to the other items too (or even better move it to a common parent and reduce duplicated code but only if you feel like doing that).